bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/transactions/TransactionResourceManager.java

```diff
@@ -618,4 +617,9 @@ public Object getTransactionRecord(BArray xid) {
             return null;
         }
     }
+
+    private void logError(String message, Exception err) {
+        outStream.println(LocalDateTime.now() + "ERROR\t" + message + "\n");
+        err.printStackTrace();
```

Comment:
Shall we replace with  out.println(Arrays.toString(e.getStackTrace()))

Reply:
Got it. We will lose the formatting though, hope that is ok.